llvm/lib/Passes/PassRegistry.def

```diff
@@ -22,7 +22,7 @@ MODULE_ANALYSIS("callgraph", CallGraphAnalysis())
 MODULE_ANALYSIS("collector-metadata", CollectorMetadataAnalysis())
 MODULE_ANALYSIS("ctx-prof-analysis", CtxProfAnalysis())
 MODULE_ANALYSIS("dxil-metadata", DXILMetadataAnalysis())
-MODULE_ANALYSIS("dxil-resource-binding", DXILResourceBindingAnalysis())
```

Comment:
Not sure that it matters but it could be argued that this change invalidates the NFC status of this PR.

It technically changes the CLI API for the shader compiler but only for explicitly listing the pass names.